linkis-computation-governance/linkis-entrance/src/main/scala/org/apache/linkis/entrance/interceptor/impl/CommentInterceptor.scala

```diff
@@ -70,8 +70,32 @@ trait CommentHelper {
 object SQLCommentHelper extends CommentHelper {
   override val commentPattern: Regex = """\s*--.+\s*""".r.unanchored
   private val comment = "(?ms)('(?:''|[^'])*')|--.*?$|/\\*.*?\\*/|#.*?$|"
+  private val comment_sem = "(?i)(comment)\\s+'([^']*)'"
   private val logger: Logger = LoggerFactory.getLogger(getClass)
 
+  def replaceComment(code: String): String = {
+    try {
```

Comment:
better to add junit test case for this new function

Reply:
ok，I do